x/launch/keeper/msg_create_chain.go

```diff
@@ -50,7 +50,11 @@ func (k msgServer) CreateChain(goCtx context.Context, msg *types.MsgCreateChain)
 		}
 	}
 
+	chain, _ := k.GetChain(ctx, id)
+	err = ctx.EventManager().EmitTypedEvent(&types.EventChainCreated{
```

Comment:
In `campaign`, a chain for the mainnet is also created by calling `CreateNewChain` when sending `MsgIninitializeMainnet`

Maybe we could have this event in `CreateNewChain` to catch everything? Or we can eventually consider a `MainnetInitialized` event